.travis.yml

```diff
@@ -10,6 +10,18 @@ env:
   - TOXENV=django111-drf38
   - TOXENV=django111-drf39
   - TOXENV=django111-drflatest
+  - TOXENV=django20-drf37
+  - TOXENV=django20-drf38
```

Comment:
Do we still need the DRF 3.7 and 3.8 tests?  Nearly all of our services have already upgraded to at least 3.9.4, and the others should before upgrading this.